bitnami/minio/templates/provisioning-job.yaml

```diff
@@ -106,6 +106,11 @@ spec:
               mc admin bucket quota {{ $minioAlias }}/{{ $bucket.name }} --{{ .type }} {{ .size }};
               {{- end }}
 
+              {{- if (or ((empty $bucket.withLock)) (not $bucket.withLock)) }}
```

Comment:
I assume this line is here because versioning is enabled by default in object locking buckets. This, however, prevents users from disabling versioning on such object locking enabled buckets. 

Now, I'm not a MinIO expert, but reading through the [documentation](https://docs.min.io/docs/minio-bucket-object-lock-guide.html) I think it doesn't make any sense to disable versioning on object locking buckets (or if it's even possible), so I'm fine with this. However, I wanted to know your thoughts on this too to double check.

Reply:
Yes that's why I gave that check. I also think that it shouldn't (or wouldn't) be possible to disable versioning on buckets with locking enabled.